### DISCUSSION TITLE
How to connect two OFFRFobjects to simulate bolted connection?

### DISCUSSION THREAD

**Discussion by qiushi-Lee** [jgerstmayr/EXUDYN#67] [General]
As the topic, I want to build a nonlinear bolted relationship between two OFFRFobjects for their dynamic response, but I don't know which joint or connection could realize this purpose.

**Comment by jgerstmayr** [comment]
You can use a `RigidBodySpringDamper`. You attach markers to the `ObjectFFRFreducedOrder`, using `MarkerSuperElementRigid` attached to them. 
In `RigidBodySpringDamper`, you can also use user-functions to define the nonlinear {force-torque} to {displacement-rotation} relations.
